app/helpers/carts_helper.rb

```diff
@@ -0,0 +1,15 @@
+module CartsHelper
+  def initialize_session
+    session[:carts] ||= {}
+  end
+
+  def total_cart
+    session[:carts].values.reduce do |one, two|
+      one + two
+    end
```

Comment:
```suggestion
    session[:carts].values.reduce(:+)
```
Sửa như này có được không em